### ISSUE TITLE
<RichTextInput> doesn't display validation errors

### ISSUE THREAD

**Issue by fzaninotto** [marmelab/react-admin#207]
With the following edition form:

```js
export const PostEdit = (props) => (
    <Edit title={<PostTitle />} {...props}>
        <DisabledInput label="Id" source="id" />
        <RichTextInput source="body" validation={{ required: true }} />
    </Edit>
);
```

It's impossible to edit a post with no `body` - and there are no explanations.

**Comment by fzaninotto** [comment]
Moved to https://github.com/marmelab/aor-rich-text-input/issues/1

**Comment by djhi** [comment]
Fixed by #1916
